src/index.js

```diff
@@ -1,2 +1,58 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...chlidren) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  chlidren.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
+  });
+
+  return element;
+}
+
+function render({ count = 0 }) {
+  function handleClick() {
+    render({ count: count + 1 });
+  }
+
+  function handleClickNumber(value) {
+    render({ count: value });
+  }
+
+  const element = (
+    <div id="hello" className="greeting">
+      <p>hell worlod</p>
+      <p>
+        <button type="button" onClick={() => handleClick()}>
```

Comment:
```suggestion
        <button type="button" onClick={ handleClick}>
```

이렇게 함수만 전달하는 형태로도 사용이 가능해요!

Reply:
밑에 버튼과 같은 컨벤션을 유지하려고 `() => handleClick()`을 사용했는데, 다음부터는 인자가 없을경우에 함수만 전달하는 방향으로 가겠습니다!